src/Symfony/Component/Messenger/Command/FailedMessagesRemoveCommand.php

```diff
@@ -61,18 +67,36 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $shouldForce = $input->getOption('force');
         $ids = (array) $input->getArgument('id');
-        $shouldDisplayMessages = $input->getOption('show-messages') || 1 === \count($ids);
-        $this->removeMessages($failureTransportName, $ids, $receiver, $io, $shouldForce, $shouldDisplayMessages);
+        $shouldDeleteAllMessages = $input->getOption('all');
 
-        return 0;
-    }
+        $idsCount = \count($ids);
+        if (!$shouldDeleteAllMessages && !$idsCount) {
+            throw new RuntimeException('Message ids must be specified. If you want to remove all messages, use the "--all" option.');
```

Comment:
we try to not use "must", neither in doc nor in any messages

```suggestion
            throw new RuntimeException('Please specify at least one message id. If you want to remove all messages, use the "--all" option.');
```

Reply:
Got it :+1: